op-node/rollup/driver/state.go

```diff
@@ -475,11 +475,6 @@ func (s *Driver) StartSequencer(ctx context.Context, blockHash common.Hash) erro
 	if !s.driverConfig.SequencerEnabled {
 		return errors.New("sequencer is not enabled")
 	}
-	if isLeader, err := s.sequencerConductor.Leader(ctx); err != nil {
```

Comment:
Interesting, do you know what’s the sequence of events for the race condition?
It is a little bit weird because in order for op-conductor to issue start sequencer request, it has to be the leader, thus the check should return true

Also conductor side has retry logic, which will re-queue the start sequencer step and try it again.
I’d imagine at some point in time the leader status would return true and allow sequencer to start.
This makes me feel like it could be some bug in the conductor rpc conductor_Leader

Reply:
I could see op-conductor issuing a start for seq 1, then switching to seq 2 as the active and issuing a second start command. It seems correct for seq 1 to fail to start. I'm not sure why seq 2 would not start in this case.